teacher_class.rb

```diff
@@ -0,0 +1,42 @@
+
+class Teacher
+
+attr_reader :age, :salary, :phase, :target_raise
+attr_accessor :name
+
+	def initialize(options={})
+		@age = options.fetch(:age, 0)
+		@name = options.fetch(:name, "")
+		@phase = 3
+	end
+
+	def set_phase(num)
+    	@phase = num
+    	"Cool, I've always wanted to teach phase #{num}!"
+  	end
+
+  	#Not enough time to add this method to parent class Teacher
+  # 	def set_performance_rating(rating)
+  #   
+  #   response = ""
+  #   if rating > 90
+  #     response = "Yay, I'm a great employee!"
+  #     receive_raise(@target_raise)
+  #   else
+  #     response += "Oh, well -- thanks to this actionable, specific, and kind "
+  #     response += "feedback, I'll do better next time."
+  #   end
+  #   response
+  # end
+
+  	def salary=(new_salary)
+   		puts "This better be good!"
+    	@salary = new_salary
+  	end
+
+  	def receive_raise(raise)
+    	@salary += raise
+  	end
+ end
```

Comment:
Watch your indentation. I know indentation doesn't always seem important, but if your code is poorly indented in a job setting, your code will not be accepted. It also makes you look unprofessional. Please make an effort to follow style guides.

Make sure to set sublime (or whatever text editor you're using) to have 2-space tabs, since that's what we use in the files we give you, and we like consistency (also, sometimes if you have tabs in some places and spaces in others, it can look right to you, but then look wrong on github).

http://stackoverflow.com/questions/9474090/how-do-i-force-sublime-text-2-to-indent-two-spaces-per-tab